debian/copyright

```diff
@@ -3,11 +3,11 @@ Upstream-Name: step
 Source: https://github.com/smallstep/cli
 
 Files: *
-Copyright: 2018 Smallstep Labs, Inc.
-License: MIT
+Copyright: 2019 Smallstep Labs, Inc.
+License: Apache
```

Comment:
@dopey the debian docs say `Apache` for APL-1.0 & APL-2.0. Can you take a look too just to be sure this will work? Thanks